Tasks/Maven/mavencodeanalysis.ts

```diff
@@ -8,8 +8,8 @@ import trm = require('vsts-task-lib/toolrunner');
 
 // Lowercased names are to lessen the likelihood of xplat issues
 import pmd = require('./mavenpmd');
-import ar = require('./analysisresult');
-import ma = require('./moduleanalysis');
+import {AnalysisResult} from './analysisresult';
+import {ModuleAnalysis} from './moduleanalysis';
```

Comment:
This change is a Typescript-specific syntax which looks much nicer - all class references below have changed to be more brief and readable.